test/SparqlParserTest.cpp

```diff
@@ -359,6 +358,9 @@ TEST(ParserTest, testParse) {
     // are implemented
     /*
     {
+     // C++ exception with description "ParseException, cause:
+     // Expected a token of type AGGREGATE but got a token of
+     // type RDFLITERAL (() in the input at pos 373 : (YEAR(?year))
```

Comment:
Yes, we currently don't support YEAR(...) and functions like YEAR() in the GROUP BY